script.js

```diff
@@ -84,21 +83,21 @@ function compareAndRender(d){
     if(leftEmoji.diameter_meters == rightEmoji.diameter_meters){
       document.getElementById('left-viz').style = "font-size : 6em";
       document.getElementById('right-viz').style = "font-size : 6em";
-      document.getElementById('info').innerHTML = leftEmoji.emoji + " is the same size as " + rightEmoji.emoji;
+      document.getElementById('info').innerHTML = `${leftEmoji.emoji} is the same size as ${rightEmoji.emoji}`;
```

Comment:
Using `+` to concat strings is fine, but I prefer to use Template literals because it is intuitive and could be helpful avoid bugs.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals